### ISSUE TITLE
Using arrows as in cookbook does not seem to work

### ISSUE THREAD

**Issue by bd339** [guicho271828/eazy-gnuplot#41]
From the arrow example in the cookbook, I would expect this code to work:

```
(eazy-gnuplot:with-plots (*standard-output* :persist t)
      	(eazy-gnuplot:gp-setup :terminal '(wxt)
      			       :arrow
      			       '(from 0 #\, 0 to 0 #\, 5 nohead))
      	;;plotting code here
        )
```

But it fails with an inexhaustive match error. I was able to get it working by adding a character case to `gp-quote`.

Is this no longer the way you are supposed to make arrows?

**Comment by guicho271828** [comment]
Thank you for pointing this out, I welcome your pull request

**Comment by guicho271828** [comment]
Closed with #43 : all fixed now.
